widget/menu.go

```diff
@@ -18,6 +18,66 @@ type Menu struct {
 	OnDismiss   func()
 	activeItem  *menuItem
 	customSized bool
+	index       int
+}
+
+func (m *Menu) Defocus() {
```

Comment:
I don't think all of these APIs should be public.

Reply:
I agree, but this was the only way I found to implement access from internal/driver/glfw/menu_bar_item. Is there any other way?

Reply:
If the menus are focusable then the usual focus methods should work - I’m not sure I understand what things the menu bar needed to signal into menus.